doc/concepts/decor.rst

```diff
@@ -166,7 +167,24 @@ Take for example the built-in mechanism for passive leaky dynamics:
 
 .. _cablecell-ions:
 
-4. Ion species
+.. _cablecell-scaled-mechs:
+
+4. Scaled mechanisms
+~~~~~~~~~~~~~~~~~~~~~
+:ref:`Inhomogenous expressions  <labels-iexpr>` can be applied to mechanisms, to scale parameters on a cable cell.
+These expressions are evaluated at mid-points of the internal cable partition of the cell and multiplied with the set parameter value.
```

Comment:
Finally his could be made clearer and best split into two sentences.
1. These expressions yield a _scaling factor_ that can be applied to a base value of a parameter (currently only density mechanisms).
2. Example
3. The point of evaluation is the mid point...

Since 3. is a important detail, but does not add to the understanding  of the concrete use.

Reply:
I've reformulated the text and changed the example to use a distance expression.